### ISSUE TITLE
CAT on 10.13.5

### ISSUE THREAD

**Issue by jdzmfg** [dokterdok/Continuity-Activation-Tool#436]

**Comment by finbarrcnaipe** [comment]
Just installed 10.3.5:

Reboot to Recovery, disable SIP, reboot, uninstall CAT, reboot, re-install CAT, reboot, reboot to Recovery, Re-Enable SIP "csr enable --without kext", reboot...   All is working.

During uninstall, CAT has not been able to restore original drivers from recovery for awhile now. So I'm not exactly sure what is going on with the uninstall.    Last time I did this (10.3.4), uninstall appeared to complete, but when re-installing CAT indicated it was already active.   In which case I checked to make sure it was working...which it was.   In any event, w/10.3.5 today the uninstall seemed to "take" (without recovery drivers), and reinstall was required and completed successfully.   The final outcome is CAT seems to be working fine and now my Continuity Activation check appears to be working again on reboots...so I'm happy about that...
